tests/foreman/ui/test_hostgroup.py

```diff
@@ -180,16 +296,18 @@ def test_positive_check_activation_keys_autocomplete(self):
 
         @Steps:
 
-        1. Create two content views A & B
-        2. Publish both content views
-        3. Create an activation key pointed to view A in Library
-        4. Create an activation key pointed to view B in Library
-        5. Go to the new Hostgroup page, select content view B & Library,
-            click on the activation key tab and click in the input box
+            1. Create two content views A & B
+            2. Publish both content views
+            3. Create an activation key pointed to view A in Library
+            4. Create an activation key pointed to view B in Library
+            5. Go to the new Hostgroup page, select content view B & Library,
+                click on the activation key tab and click in the input box
```

Comment:
Lol. I have no idea when I did that. Anyway, if travis is not failing on that, let's skip that